lib/ui/src/modules/ui/components/stories_panel/index.js

```diff
@@ -50,17 +50,23 @@ class StoriesPanel extends Component {
           onChange={text => onStoryFilter(text)}
         />
         <div style={scrollStyle}>
-          {hierarchyContainsStories(storiesHierarchy) ? (
-            <Stories {...pick(this.props, storyProps)} />
-          ) : null}
+          {storiesHierarchies.map(
```

Comment:
worth extracting it to function already.

Reply:
extracted the map it to separate render method.
extracted the sub header to stories_tree/ as `tree_header`